app/src/main/java/grinnell/appdev/edu/lyles/fragments/MenuTabFragment.java

```diff
@@ -0,0 +1,108 @@
+package grinnell.appdev.edu.lyles.fragments;
+
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+
+import grinnell.appdev.edu.lyles.MenuTabListAdapter;
+import grinnell.appdev.edu.lyles.R;
+
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.DRINKS_INDEX;
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.HOT_FOOD_INDEX;
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.SNACKS_INDEX;
+
+/**
+ * A fragment to display available drinks to users
+ */
+public class MenuTabFragment extends Fragment {
+
+    public final int position;
+
+    public MenuTabFragment() {
+        this.position = -1;
+    }
+
+    @SuppressLint("ValidFragment")
```

Comment:
Aha! As mentioned above, I think the linter is complaining b/c you don't have an empty constructor which is required for Fragments. Fragments require, by default, an empty constructor so they can be re-instantiated when the hosting `Activity`'s state is being restored. It'll nag you to create a "Valid Fragment" b/c it can't use the given constructor.